### ISSUE TITLE
Move bower components to outside resources/static

### ISSUE THREAD

**Issue by grtjn** [rjrudin/slush-marklogic-spring-boot#45]
To make migration of ui code from slush-marklogic-node easier..

**Comment by grtjn** [comment]
Should require only following changes..

- Change `_bowerrc` to point to: `src/main/bower_components`
- Change `src/main/resources/bower_components` to `src/main/bower_components` in `_gitignore`
- Add this to `gulpfile.js`:

```
gulp.task('copy-bower', [], function() {
  return gulp.src('src/main/bower_components/**/*')
  .pipe(gulp.dest(staticDir + '/bower_components/'));
});
gulp.task('copy-app', ['clean-app', 'copy-bower'], function() {
```

Once that is in place, it will no longer be necessary to manually fix the imports in `ui/styles/main.less` when copying over from slush-marklogic-node..

**Comment by rjrudin** [comment]
This may require some Spring Boot trickery - Spring Boot expects to find static resources under src/main/resources, though there are other places it looks too, and it may be possible to customize that list.

**Comment by grtjn** [comment]
Yes, that is why you need to add the gulp task to 'copy-bower'..

**Comment by grtjn** [comment]
And then all is well, no spring boot trickery..
